### DISCUSSION TITLE
翻译ontology101

### DISCUSSION THREAD

**Discussion by lidingpku** [cnschema/cnschema#65] [task]
目标：
这篇文章对schema设计有很重要的指导意义，希望大家参与进来翻译成中文，帮助更多的cnSchema领域扩展者解决schema设计的疑惑。

结果
* 招募志愿者
* 在cnschema发布中文版

资源：
http://ksl.stanford.edu/people/dlm/papers/ontology-tutorial-noy-mcguinness-abstract.html

**Comment by iblisylvie** [comment]
我很有兴趣帮忙翻译。但是不太清楚操作流程是什么，是把译文直接黏贴在这里吗？

**Comment by lidingpku** [comment]
https://github.com/cnschema/cnschema/blob/master/misc/ontology101cn.md

I'm editing it and anyone can fork and generate a pull request

**Comment by sunerye** [comment]
求参与

**Comment by lidingpku** [comment]
做了第一轮翻译：
可以直接在GoogleDoc上提建议（或者参与编辑）
https://docs.google.com/document/d/14KyP49qiYG7SgSM-3bedQuTif5ewzGfimVLnRx6Mo-0/edit#

也可以在github上提交PULL REQUEST 
https://github.com/cnschema/cnschema/blob/master/misc/ontology101cn.md

还需要
* 增补原文的图片（并且增加一些中文注释）
* 修订翻译中专业术语的一致性问题
* 发现翻译错误，调整文章语言的流畅性
* 提出文章内容的其他期望（针对中文知识图谱领域建模，还有哪些问题亟待解决），下一版应该如何写

**Comment by lidingpku** [comment]
资料 
进口葡萄酒相关术语翻译规范SB/T11122-2015
Norm of Terminology Translation of Imported Wines
http://www.xmjlgh.com/fgybz/downloadAcc.asp?filename=80.pdf

**Comment by lxylilly** [comment]
完成原文图片增补和翻译

**Comment by lidingpku** [comment]
后续工作

将最新版（含图片，https://docs.google.com/document/d/14KyP49qiYG7SgSM-3bedQuTif5ewzGfimVLnRx6Mo-0/edit  ）更新到cnschema官网发布 https://github.com/cnschema/cnschema/blob/master/misc/ontology101cn.md  
   *  文字的校验
   *  排版更新
   *  贴图片
